fastchat/serve/monitor/monitor.py

```diff
@@ -1052,6 +1052,10 @@ def build_leaderboard_tab(
                 build_full_leaderboard_tab(
                     elo_results_text, model_table_df, model_to_score
                 )
+            with gr.Tab("Copilot Arena Leaderboard", id=5):
+                from fastchat.serve.monitor.copilot_arena import build_copilot_arena_tab
+
+                build_copilot_arena_tab()
```

Comment:
can we make this tab optional and default not showing?  Otherwise people who do not have access to `COPILOT_ARENA_LEADERBOARD_URL` will error out.

We can add an argument to the `build_leaderboard_tab` function

Reply:
What if for simplicity we just try-catch and log it if it doesn't get built?